pkgs/stdenv/cross/default.nix

```diff
@@ -1,35 +1,48 @@
-{ lib, allPackages
+{ lib
 , system, platform, crossSystem, config
 }:
 
-rec {
-  vanillaStdenv = (import ../. {
-    inherit lib allPackages system platform;
+let
+  bootStages = import ../. {
+    inherit lib system platform;
     crossSystem = null;
-    # Ignore custom stdenvs when cross compiling for compatability
+    # Remove config.replaceStdenv to ensure termination.
```

Comment:
No need to change this, but can you explain the termination issue when `replaceStdenv` is there?